internal/exec/util/selinux.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/coreos/ignition/v2/internal/distro"
+)
+
+const (
+	selinuxConfig       = "/etc/selinux/config"
+	selinuxFileContexts = "contexts/files/file_contexts"
+)
+
+var selinuxPolicy = ""
+
+func (ut Util) getSelinuxPolicy() (string, error) {
+	if selinuxPolicy == "" {
```

Comment:
super minor nit: take it or leave it, I'd invert this if and do a short return if it's already set.

Reply:
This is the classic "1. if cache is uninitialized, initialize it, 2. return cached value". I think I'd rather keep the current structure to make that obvious?